docs/getting_started.md

```diff
@@ -30,62 +32,105 @@ If you see any changes between the two feel free to create a
 [pull request](https://github.com/google/flax/compare)
 to synchronize this Colab with the actual example.
 
++++ {"id": "nwJWKIhdwxDo"}
+
+## 1. Install Flax
+
 ```{code-cell}
----
-colab:
-  base_uri: https://localhost:8080/
-id: FOYfP6vf2FGx
-outputId: 65ba547d-59df-4fc0-9578-9faa2c14abe8
-tags: [skip-execution]
----
-# Check GPU
-!nvidia-smi -L
+:id: bb81587e
+:tags: [skip-execution]
+
+!pip install -q flax
 ```
 
-+++ {"id": "VSaA8Mif6srP"}
++++ {"id": "bbc9b4f2"}
 
-## 1. Imports
+## 2. Imports
 
 Import JAX, [JAX NumPy](https://jax.readthedocs.io/en/latest/jax.numpy.html),
 Flax, ordinary NumPy, and TensorFlow Datasets (TFDS). Flax can use any
 data-loading pipeline and this example demonstrates how to utilize TFDS.
 
 ```{code-cell}
-:id: inJ9bV636dRx
-:tags: [skip-execution]
-
-!pip install -q flax
-```
-
-```{code-cell}
-:id: Z7MuGFB16E8m
-
+---
+executionInfo:
+  elapsed: 54
+  status: ok
+  timestamp: 1671500846075
+  user:
+    displayName: Marcus Chiam
+    userId: '17531616275590396120'
+  user_tz: 300
+id: a9633134
+---
 import jax
 import jax.numpy as jnp                # JAX NumPy
 
 from flax import linen as nn           # The Linen API
 from flax.training import train_state  # Useful dataclass to keep train state
+from flax import struct                # Flax dataclasses
 
 import numpy as np                     # Ordinary NumPy
 import optax                           # Optimizers
+import tensorflow as tf                # Tensorflow to operate on TFDS
 import tensorflow_datasets as tfds     # TFDS for MNIST
+import matplotlib.pyplot as plt        # Visualization
+```
+
++++ {"id": "b529fbef"}
+
+## 3. Loading data
+
+Define a function that loads and prepares the MNIST dataset and converts the
+samples to floating-point numbers.
+
+```{code-cell}
+---
+executionInfo:
+  elapsed: 52
+  status: ok
+  timestamp: 1671500846272
+  user:
+    displayName: Marcus Chiam
+    userId: '17531616275590396120'
+  user_tz: 300
+id: bRlrHqZVXZvk
+---
+def get_datasets():
+  """Load MNIST train and test datasets into memory."""
+  train_ds = tfds.load('mnist', split='train')
+  test_ds = tfds.load('mnist', split='test')
+
+  train_ds = train_ds.map(lambda sample: {'image': tf.cast(sample['image'], tf.float32) / 255., 'label': sample['label']}) # normalize train set
+  test_ds = test_ds.map(lambda sample: {'image': tf.cast(sample['image'], tf.float32) / 255., 'label': sample['label']}) # normalize test set
+
+  return train_ds, test_ds
 ```
 
-+++ {"id": "d0FW1DHa6cfH"}
++++ {"id": "7057395a"}
 
-## 2. Define network
+## 4. Define network
 
 Create a convolutional neural network with the Linen API by subclassing
 [Module](https://flax.readthedocs.io/en/latest/flax.linen.html#core-module-abstraction).
```

Comment:
Nit:

In other guides we are/started using "Flax Module" since "module/Module" is a common word.

Maybe here:

```suggestion
[Flax Module](https://flax.readthedocs.io/en/latest/flax.linen.html#core-module-abstraction).
```

And then repeat this, which can help new users.